File: Get Info/app/src/main/java/com/harivansh/gitinfo/api/RepoApi.java
package com.harivansh.gitinfo.api;

import com.harivansh.gitinfo.model.Repo;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface RepoApi {

    @GET("users/{user}/repos")
    Call<List<Repo>> listRepos(@Path("user") String user);
}

File: Get Info/app/src/main/java/com/harivansh/gitinfo/api/RepoApi.kt
package com.harivansh.gitinfo.api

import com.harivansh.gitinfo.model.Repo
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface RepoApi {
    @GET("users/{user}/repos")
    fun listRepos(@Path("user") user: String?): Call<List<Repo>>
}